workflow/qa-steps.md

```diff
@@ -0,0 +1,35 @@
+# QA steps
+
+In this section you will find a list of steps to take when reviewing PRs, in order to catch unforeseen regressions.
+
+## Adding content
+
+- If adding a new page to a site, check that it has been added to the navigation, sitemap and breadcrumbs (where applicable)
+- If applicable, check that the new content's copy matches the relevant copydoc.
+- Conversely, update the copydoc if it is out of date with the content.
```

Comment:
I don't think this is the responsibility of the reviewer. "Check that the new content matches the relevant copydoc" should cover it - as in, let the person who did the work know of the discrepancy. Resolving the discrepancy is best done by the person doing the actual work.